pkg/server/router.go

```diff
@@ -59,6 +62,11 @@ func (r *Router) Routes(h router.Handlers) http.Handler {
 	m.Path("/v1/harvester/backuptarget/healthz").Methods("GET").Handler(btHealthyHandler)
 	// --- END of preposition routes ---
 
+	// This is for manually testing the recovery handler below
+	m.HandleFunc("/v1/harvester/dont-panic", func(_ http.ResponseWriter, _ *http.Request) {
+		panic("Do you know where your towel is?")
+	})
+
```

Comment:
Do we need to remove this line if we're already tested it?

Reply:
I wasn't sure about that. I tested it just now while updating this PR, so I know it works, and it is a pretty silly endpoint :-) OTOH, if we remove it, there's no way to manually test the recovery handler in future (think: regression testing).

Reply:
I come up with a idea, we just need to write the mock server test case. I think it should work.

```go
m := mux.NewRouter()
m.Use(recoveryMiddleware)

// put your dont-panic here
m.HandleFunc("/v1/harvester/dont-panic", func(_ http.ResponseWriter, _ *http.Request) {
    panic("Do you know where your towel is?")
})

// new request
req, err := http.NewRequest("GET", "/v1/harvester/dont-panic", nil)

// the use httptest to mock server
responseWriter := httptest.NewRecorder()
m.ServeHTTP(responseWriter, req)

// then check responseWriter
```

Reply:
AIUI that'll test the recoveryMiddleware itself works (good!) but won't provide any guarantee that the middleware is actually used in our router at runtime.

Reply:
Okay, let's keep it!

Reply:
OK, thanks for the review!